frontend/src/lib/components/ui/storeCard/storeCard.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+	import * as Card from '$lib/components/ui/card';
+
+  export let name: string = "";
+  export let description: string = "";
+  export let picture: string = "";
+  export let address: string = "";
```

Comment:
maybe fill something default to know miss fill

Reply:
fix